lib/tokens/tokens.go

```diff
@@ -0,0 +1,42 @@
+package tokens
+
+type Token string
+
+const (
+	OMNI Token = "OMNI"
+	ETH  Token = "ETH"
+)
+
+var (
+	coingeckoIDs = map[Token]string{
+		OMNI: "omni-network",
+		ETH:  "ethereum",
+	}
+)
+
+func (t Token) String() string {
+	return string(t)
+}
+
+func (t Token) CoingeckoID() string {
```

Comment:
If that's the same as `String` do we really need it here?

Reply:
They're different. Ex "ETH" and "ethereum"